planner/core/testdata/join_reorder_suite_out.json

```diff
@@ -2533,60 +2533,60 @@
           "  │   └─Selection 9990.00 cop[tikv]  not(isnull(test.t3.b))",
           "  │     └─TableFullScan 10000.00 cop[tikv] table:t3, partition:p2 keep order:false, stats:pseudo",
           "  └─HashJoin(Probe) 58476.62 root  inner join, equal:[eq(test.t.b, test.t2.b)]",
-          "    ├─HashJoin(Build) 46781.30 root  left outer join, equal:[eq(test.t.a, test.t1.a)]",
```

Comment:
Expected, just update HashJoin build and probe sides since model2 takes row-size into consideration when calculating HashJoin costs.